code/modules/reagents/chemistry/reagents/pyrotechnic.dm

```diff
@@ -43,6 +45,7 @@
 	reagent_state = SOLID
 	color = "#673910" // rgb: 103, 57, 16
 	process_flags = ORGANIC | SYNTHETIC
+	taste_message = "metal"
```

Comment:
"Rusty metal". Iron oxide is one of the elements used and as a reagent should be there if stabilized